arm9/src/install.c

```diff
@@ -398,16 +400,39 @@ bool install(char* fpath, bool systemTitle)
 			goto error;
 		}
 
+		//patch dev titles to system titles on SysNAND.
+		//
+		//software released through the TWL SDK usually comes as a TAD and an SRL
+		//things like NandFiler have a TAD with a TID of 0x00030015 and an SRL with 0x00030011
+		//the TAD is the installable version, so I'm assuming that 0x00030015 is what the console wants to see on NAND
+		//this changes the SRL TID accordingly
+		//not entirely sure why there's even any difference. I think the installed TAD and SRL the same as each other (minus the TID)
+		if(!sdnandMode && h->tid_high == 0x00030011)
+		{
+			h->tid_high = 0x00030015;
+			fixHeader = true;
+		}
+
 		//offer to patch system titles to normal DSiWare on SysNAND
-		if(!sdnandMode && h->tid_high != 0x00030004)
+		if((!sdnandMode && h->tid_high != 0x00030004) && (!sdnandMode && h->tid_high != 0x00030017)) //do not allow patching home menus to be normal DSiWare! This will trigger "ERROR! - 0x0000000000000008 HWINFO_SECURE" on prototype launchers. May also cause issues on the prod versions.

```

Comment:
This if can be simplified to this:
```suggestion
		if(!sdnandMode && h->tid_high != 0x00030004 && h->tid_high != 0x00030017) //do not allow patching home menus to be normal DSiWare! This will trigger "ERROR! - 0x0000000000000008 HWINFO_SECURE" on prototype launchers. May also cause issues on the prod versions.
```